core/templates/pages/maintenance-page/maintenance-page.component.html

```diff
@@ -7,28 +7,28 @@
             <div class="navbar-header protractor-test-navbar-header float-left">
               <a class="oppia-navbar-brand-name oppia-transition-200" href="/">
                 <picture>
-                  <source type="image/webp" ng-srcset="<[getStaticImageUrl('/logo/288x128_logo_mint.webp')]>">
-                  <source type="image/png" ng-srcset="<[getStaticImageUrl('/logo/288x128_logo_mint.png')]>">
-                  <img ng-src="<[getStaticImageUrl('/logo/288x128_logo_mint.png')]>"
+                  <source type="image/webp" [srcset]="getStaticImageUrl('/logo/288x128_logo_mint.webp')">
+                  <source type="image/png" [srcset]="getStaticImageUrl('/logo/288x128_logo_mint.png')">
+                  <img [src]="getStaticImageUrl('/logo/288x128_logo_mint.png')"
                        class="oppia-logo oppia-logo-wide" alt="Oppia Home">
                 </picture>
               </a>
             </div>
-            <div ng-cloak class="navbar-header float-right">
+            <div class="navbar-header float-right">
             </div>
           </div>
         </nav>
 
         <div class="oppia-top-of-page-padding"></div>
         <div class="oppia-page-cards-container">
-          <md-card class="oppia-page-card oppia-long-text">
+          <mat-card class="oppia-page-card oppia-long-text">
             <h2>The Oppia site is temporarily unavailable.</h2>
 
             <p>
               Oppia is currently being upgraded, and the site should be up
               and running again in a few hours. Thanks for your patience!
             </p>
-          </md-card>
```

Comment:
This change, it didn't affect the padding at all?

Reply:
It did, it made the padding smaller. I assumed that was because of material though, since I didn't change any padding

Reply:
Yup, mat-card does that, see the padding values before and add them in a style tag here. I am pretty sure this will be the values (https://github.com/oppia/oppia/pull/9957/files#diff-dca0d61609f6bddf51cdb0301fa04f5bR21-R23) but check once.